Solutions/Azure Active Directory/Analytic Rules/BypassCondAccessRule.yaml

```diff
@@ -37,22 +37,23 @@ query: |
   let aadFunc = (tableName:string){
   table(tableName)
   | where ConditionalAccessStatus == 1 or ConditionalAccessStatus =~ "failure"
+  | extend d = parse_json(ConditionalAccessPolicies)
+  | mv-apply d on (
+    project ConditionalAccessPoliciesName = d.displayName, result = d.result
+    | where result == "failure"
```

Comment:
Nice update!
Can you make this a =~ to add some resiliency in case of future data source changes.

Reply:
Rosolved in https://github.com/Azure/Azure-Sentinel/pull/6706